apps/trustlab/src/payload/blocks/DonorOverview.js

```diff
@@ -0,0 +1,31 @@
+const Donors = {
+  slug: "donor-overview",
+  imageURL: "/images/cms/blocks/partners.png",
+  imageAltText: "Donors overview",
+  labels: {
+    singular: {
+      en: "Donors",
+    },
+    plural: {
+      en: "Donors",
+    },
+  },
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "partners",
+      type: "relationship",
+      relationTo: "partners",
```

Comment:
```suggestion
      name: "donors",
      type: "relationship",
      relationTo: "donors",
```
We need to keep donors (EU, etc.) separate from Partners (DWA, etc.).